### PR TITLE
Check in/72617/confirmation page updates

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Modifies the confirmation/complete page of day

## Related issue(s)

- [department-of-veterans-affairs/va.gov-team#72617](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72617)

## Testing done

- Unit
- cypress
- visual

## Screenshots

Screenshot 2024-01-17 at 2 44 20 PM

## What areas of the site does it impact?

Check-in -> day of -> complete page

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/check-in/hooks/useSendTravelPayClaim.jsx [hunk 1]
```diff
@@ -74,4 +74,4 @@ const useSendTravelPayClaim = appointment => {
   };
 };
 
-export default useSendTravelPayClaim;
+export { useSendTravelPayClaim };
```
Comment (brianseek) [inline src/applications/check-in/hooks/useSendTravelPayClaim.jsx:77]: is this just a style update?

File: src/applications/check-in/locales/en/translation.json [hunk 2]
```diff
@@ -81,7 +81,7 @@
   "if-you-traveled-from-a-different-address--helptext": "If you traveled from a different address, you can’t file a claim now. But you can file your claim online 24/7 through the Beneficiary Travel Self Service System (BTSSS). Or you can use VA Form 10-3542 to submit a claim by mail, fax, email, or in person.",
   "if-you-wait-more-than": "If you wait more than 15 minutes, tell a staff member.",
   "in-en": "in English",
-  "in-person": "In person at",
+  "in-person-at": "In person at",
```
Comment (brianseek) [inline src/applications/check-in/locales/en/translation.json:84]: why the key name change?

Comment (brianseek) [reply src/applications/check-in/locales/en/translation.json:84]: oh nevermind, I see why

File: src/applications/check-in/components/AppointmentDisplay/AppointmentListItem.jsx [hunk 3]
```diff
@@ -26,6 +26,21 @@ const AppointmentListItem = props => {
   const selectFeatureToggles = useMemo(makeSelectFeatureToggles, []);
   const { is45MinuteReminderEnabled } = useSelector(selectFeatureToggles);
 
+  const detailsAriaLabel = () => {
```
Comment (brianseek) [inline src/applications/check-in/components/AppointmentDisplay/AppointmentListItem.jsx:29]: what was the reason to re-factor this?

File: src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx [hunk 4]
```diff
@@ -129,7 +126,10 @@ const CheckInConfirmation = props => {
         pageTitle={t('youre-checked-in')}
         testID="multiple-appointments-confirm"
       >
-        <p className="vads-u-font-family--serif">{t('your-appointment')}</p>
+        <p data-testid="confirmation-message">
+          {t('well-come-get-you-from-the-waiting-room')}
```
Comment (brianseek) [inline src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx:130]: This text is different than what is in the figma file. Am I looking at the wrong thing?

I see this:
Screenshot 2024-01-17 at 3 38 01 PM

File: src/applications/check-in/day-of/pages/Confirmation/TravelPayAlert.jsx [hunk 5]
```diff
@@ -4,106 +4,125 @@ import { useTranslation, Trans } from 'react-i18next';
...
-      {travelPayClaimError ? (
-        <va-alert
-          show-icon
-          data-testid="travel-pay-message"
-          status="warning"
-          uswds
-          slim
+  if (!travelPayClaimRequested) {
+    return (
+      <va-alert show-icon data-testid="travel-pay-info-message" uswds slim>
```
Comment (brianseek) [inline src/applications/check-in/day-of/pages/Confirmation/TravelPayAlert.jsx:17]: From looking at the figma, it seems like this text is no longer wrapped in an alert.

**Review by brianseek — CHANGES_REQUESTED** [review]
just a couple of things to clear up. Did you ask Zach about these error screens? They don't make sense to me. We wouldn't have the Your appointment section, plus we wouldn't submit travel if the check-in failed, so the mix scenarios don't make sense. We can bring it up as a PL tomorrow if you haven't had the conversation yet.

Screenshot 2024-01-17 at 3 44 50 PM

**Comment by brianseek** [comment]
Maybe those screens do make sense, but we add those sections to the error page. Ether way that work still needs to be part of this PR

**Comment by benbrasso-agile6** [comment]
They're no different than what's in production. But, def can talk them through.

**Comment by benbrasso-agile6** [comment]
For reference of what I think we have in production when someone attempts to check in. https://www.sketch.com/s/a31fc97a-2d0c-4ef0-a698-7b73b773836f/p/1A228FD4-6078-4206-98D1-E26C374F342C/canvas

**Comment by brianseek** [comment]
> For reference of what I think we have in production when someone attempts to check in. https://www.sketch.com/s/a31fc97a-2d0c-4ef0-a698-7b73b773836f/p/1A228FD4-6078-4206-98D1-E26C374F342C/canvas

Yeah it took a minute for my brain to click, after I stared at them for a minute. The added headings do need to be added to that error page in this PR though.

**Comment by benbrasso-agile6** [comment]
K screenshot it when you get a chance

File: src/applications/check-in/components/AppointmentDisplay/AppointmentListItem.jsx (hunk 3 above)
Comment (wullaski) [reply src/applications/check-in/components/AppointmentDisplay/AppointmentListItem.jsx:29]: It was in the wire frames the aria label changed.

Comment (wullaski) [reply src/applications/check-in/components/AppointmentDisplay/AppointmentListItem.jsx:29]: https://www.figma.com/file/sK2vIx6giAynNqUFR0l4Wx/Production-%7C-Check-in?type=design&node-id=29-614&mode=design&t=affs9Cf0bystQVWb-0

Comment (brianseek) [reply src/applications/check-in/components/AppointmentDisplay/AppointmentListItem.jsx:29]: oh nice, thanks for the link to that view!

File: src/applications/check-in/hooks/useSendTravelPayClaim.jsx (hunk 1 above)
Comment (brianseek) [reply src/applications/check-in/hooks/useSendTravelPayClaim.jsx:77]: oh cool, I was just curious

File: src/applications/check-in/day-of/pages/Confirmation/TravelPayAlert.jsx (hunk 5 above)
Comment (wullaski) [reply src/applications/check-in/day-of/pages/Confirmation/TravelPayAlert.jsx]: fixed

File: src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx (hunk 4 above)
Comment (wullaski) [reply src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx:130]: This is odd. I see the correct text and line 130 is the correct key for it.
Screenshot 2024-01-18 at 11 45 50 AM

Comment (brianseek) [reply src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx:130]: what I screenshot is what is in figma, not what I am seeing in the app

**Comment by wullaski** [comment]
@brianseek, @benbrasso-agile6  Sorry, I missed the changes to the error page on the first pass. Those changes are in now.

**Comment by benbrasso-agile6** [comment]
Periods at the end of headings are old. All of our new designs don't use them (and we shouldn't). So, it's a slow move to getting rid of them. But, feel free to drop periods at the end of h1's if it's just one sentence. Except for when it's a question that needs a question mark.

File: src/applications/check-in/day-of/pages/Error.jsx [hunk 6]
```diff
@@ -167,16 +168,34 @@ const Error = () => {
...
-            slim
-          >
-            <div>{alert.message}</div>
-          </va-alert>
+          {alert.travelMessage ? (
+            <h2 data-testid="travel-message-heading">
+              {t('travel-reimbursement')}
+            </h2>
+          ) : (
+            <h2 data-testid="appointment-message-heading">
```
Comment (brianseek) [inline src/applications/check-in/day-of/pages/Error.jsx:176]: This needs an error type check probably too, the designs don't have this heading for errors other than ones that originate from clicking check-in

**Review by brianseek — CHANGES_REQUESTED** [review]
one little change on error page, more detail in the slack thread

**Review by brianseek — DISMISSED** [review]
Nice work on the changes and evolution of this! Might be worth having someone from UX check it out locally, either way I would grab screens of all the changed pages for them to review.

**Review by brianseek — APPROVED** [review]
UX feedback changes LGTM!

**Pull request merged**